### DISCUSSION TITLE
What are fonts supposed to look like with SDL2/SDL2_ttf?

### DISCUSSION THREAD

**Discussion by jharrison03** [dosemu2/dosemu2#2117] [General]
Attached is a screen shot of my DOSEmu2 session.  I don't know what the fonts are supposed to look like, but they've always looked this this.  Is there any further configuration needed, such as making the VGA fonts shipped with DOSEmu2 available to fontconfig (which I've done). 

![2023-10-25-081627_2048x768_scrot](https://github.com/dosemu2/dosemu2/assets/44394516/2235da81-de33-46d0-b613-ded9343b0be6)

**Comment by stsp** [comment]
Firstly, new fonts are available only
in a default setup. I am not sure your
setup is default, I see SHSUCDX and
other 3rd-party stuff.
Fonts are likely not loaded.

**Comment by jharrison03** [comment]
It's not default and I'll go through it again.

I'm just not sure if that's how they're supposed to appear. (?)

**Reply by stsp** [reply]
Try `xmode -custom-font on`
They are supposed to appear MUCH
better than on your screen.

**Comment by jharrison03** [comment]
If figured this isn't how they're supposed to look.  So, DOSEmu2 ships with two sets of fonts, one set installs under _share/dosemu/Xfonts_, and one set under _share/fonts/oldschool_

Both of these paths are in my fontconfig setup.  The 'oldschool' font I think is what I should be seeing, but I don't, but this font is available in other programs through their various font selection dialogs.

**Comment by stsp** [comment]
So have you done xmode -custom-font on?

**Comment by jharrison03** [comment]
OK, that did it.  I didn't know that should be executed from within DOSEmu.  (Was looking for 'xmode' on my system).

**Reply by stsp** [reply]
Which is why, as I said, you should
stick to the default setup. Then you'll
get all new features in time.
